vector/src/main/java/im/vector/app/features/navigation/DefaultNavigator.kt

```diff
@@ -501,7 +502,10 @@ class DefaultNavigator @Inject constructor(
     }
 
     override fun openCreatePoll(context: Context, roomId: String) {
-        val intent = CreatePollActivity.getIntent(context, CreatePollArgs(roomId = roomId))
+        val intent = CreatePollActivity.getIntent(
+                context,
+                CreatePollArgs(roomId = roomId, minOptionsCount = CreatePollViewModel.MIN_OPTIONS_COUNT)
```

Comment:
What is the point to pass `minOptionsCount` as an Activity parameter?

Reply:
I thought it was necessary to use it in `CreatePollViewState`. Fixed by using const value in ViewModel directly.